### PR TITLE
Fix migrate mirror 500 bug (#8526)

### PR DESCRIPTION
**Pull request by 6543**
Backports https://github.com/go-gitea/gitea/pull/8526 -> thanks @lunny 

will close https://github.com/go-gitea/gitea/issues/8514

### CODE REVIEW

**Review by guillep2k — APPROVED** [review]

File: models/repo.go [hunk 1]
```diff
@@ -1430,6 +1430,7 @@ func CreateRepository(doer, u *User, opts CreateRepoOptions) (_ *Repository, err
 		IsFsckEnabled:                   !opts.IsMirror,
 		CloseIssuesViaCommitInAnyBranch: setting.Repository.DefaultCloseIssuesViaCommitsInAnyBranch,
 		Status:                          opts.Status,
+		IsEmpty:                         true,
```
Comment (zeripath) [inline models/repo.go]:
```suggestion
		IsEmpty:                         !opts.AutoInit,
```

**Review by zeripath — COMMENTED** [review]
@lunny changed the commit

File: models/repo.go (hunk 1 above)
Comment (6543) [reply models/repo.go]: already noticed - just tested it out

**Comment by 6543** [comment]
@zeripath o the rename was you ... you can review it now :)

**Review by lunny — APPROVED** [review]

**Pull request merged**